DOCS-1407: Add viam module generator / README template to create page (#2473)
viamrobotics/docs: docs/registry/create/_index.md

{{% /tab %}}
{{< /tabs >}}

## (Optional) create a README

To provide usage instructions for any modular resources in your module, you should create a <file>README.md</file> file following this template:

{{% expand "Click to view template" %}}

> Strings surrounded by `<%= %>` indicate variables that you should replace with the appropriate values.

Reviewer: ```suggestion
Strings of the form `<INSERT X>` indicate variables that you should replace with the appropriate values.
```

{{% /tab %}}
{{% /tabs %}}

## (Optional) create a README

To provide usage instructions for any modular resources in your module, you should create a <file>README.md</file> file following this template:

{{% expand "Click to view template" %}}

> Strings surrounded by `<%= %>` indicate variables that you should replace with the appropriate values.

Reviewer: I think it makes sense to indicate variables that need replacing in the same way we do across code samples in the docs.

If you follow [the instructions to generate module scaffolding for a Python module](#optional-generate-your-python-module-scaffolding), the template file is generated for you with the environment variables filled in.

````md

> # <INSERT NAME> modular service

Author: ```suggestion
# <INSERT NAME> modular resource
```

This module implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api) in a <%= model %> model.
With this model, you can...

## Requirements

_Add instructions here for any requirements._

> ```bash

Reviewer: I'm not sure if you need to escape these?

```

> ````

Reviewer: what are these 4 here?
